index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Burger menu</title>
+    <link rel="stylesheet" href="style.css"> 
+</head>
+<body>
+
+    <button class="menu-button">
```

Comment:
Вийшло дуже круто!
Може  я б ще додав тип, та чи то текст, чи то [лейбл](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label) для читалок.
```suggestion
    <button class="menu-button" type="button" aria-label="Close">
```
Ну і ще може зв'язав з меню
```suggestion
    <button class="menu-button" type="button" aria-label="Close" aria-controls="navigation" aria-expanded="false">
```
і додати 
```html
<nav class="menu" id="navigation"> 
```

- https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded